webui/react/src/components/kit/RadioGroup.tsx

```diff
@@ -10,12 +10,14 @@ import useResize from 'hooks/useResize';
 
 import css from './RadioGroup.module.scss';
 
-interface Props {
+interface Props extends Omit<RadioGroupProps, 'onChange' | 'options'> {
```

Comment:
Some props are overlapped with `RadioGroupProps`

Reply:
Then `onChange` doesn't have to be omitted??
Anyways what i meant is that we should follow the other components. Most of UIKit components don't extend props with antd props